terraform/examples/only_daos_server/variables.tf

```diff
@@ -155,11 +155,19 @@ variable "server_pools" {
   description = "If provided, this module will generate a script to create a list of pools. pool attributes have to be specified in a format acceptable by [dmg](https://docs.daos.io/v2.0/admin/pool_operations/) and daos."
   default     = []
   type = list(object({
-    pool_name  = string
-    pool_size  = string
-    containers = list(string)
-    })
-  )
+    name       = string
+    size       = string
+    tier_ratio = number
+    acls       = list(string)
+    properties = list(string)
+    containers = list(object({
+      name            = string
+      type            = string
+      acls            = list(string)
+      properties      = list(string)
+      user_attributes = list(string)
+    }))
+  }))
```

Comment:
https://github.com/daos-stack/google-cloud-daos/pull/40/files#diff-636c2c61569c377aa9d379a373d2cd9ae6b72dc0ef8d9c96a983f04c38d2142c  lets just have 1 "Type" delectation for Pools having 2 places of the same thing will be confusing in a very sort time.